sw/device/silicon_creator/rom/e2e/keymgr/rom_e2e_keymgr_init_test.c

```diff
@@ -23,12 +27,51 @@
 
 OTTF_DEFINE_TEST_CONFIG();
 
+static dif_keymgr_t keymgr;
+static dif_otp_ctrl_t otp_ctrl;
+static dif_rstmgr_t rstmgr;
 static uint32_t otp_state[kHmacDigestNumWords + 4] = {0};
 
+static void print_otp_sw_cfg_digests(void) {
+  uint64_t creator_digest, owner_digest = 0;
+  CHECK_DIF_OK(dif_otp_ctrl_get_digest(
+      &otp_ctrl, kDifOtpCtrlPartitionCreatorSwCfg, &creator_digest));
+  CHECK_DIF_OK(dif_otp_ctrl_get_digest(
+      &otp_ctrl, kDifOtpCtrlPartitionOwnerSwCfg, &owner_digest));
+  LOG_INFO("CreatorSwCfg Digest: 0x%08x%08x",
+           (uint32_t)((creator_digest >> 32) & 0x00000000FFFFFFFF),
+           (uint32_t)(creator_digest & 0x00000000FFFFFFFF));
+  LOG_INFO("OwnerSwCfg Digest:   0x%08x%08x",
+           (uint32_t)((owner_digest >> 32) & 0x00000000FFFFFFFF),
+           (uint32_t)(owner_digest & 0x00000000FFFFFFFF));
```

Comment:
FYI, the cast to `uint32_t` makes the masking operation unnecessary:
```
  LOG_INFO("CreatorSwCfg Digest: 0x%08x%08x",
           (uint32_t)(creator_digest >> 32),
           (uint32_t)(creator_digest));
```

Reply:
fixed.